### ISSUE TITLE
Missing attribute on GenericProvider

### ISSUE THREAD

**Issue by iromli** [GluuFederation/gluu-flask#76]
On creating new node with generic type, an error occurs caused by missing attribute in GenericProvider object.

```
   if provider.driver == 'generic' and provider.used == 'no':
AttributeError: 'GenericProvider' object has no attribute 'used'
```

The problematic line is found [here](https://github.com/GluuFederation/gluu-flask/blob/0e21e062fcabbb6262be4868f40733bf5aed3268/gluuapi/machine/machine.py#L145).

My guess is we can use `GenericProvider.is_in_use` method as a replacement.

**Comment by iromli** [comment]
@shouro any thoughts?

**Comment by iromli** [comment]
Fixed in 6afc454e8b37625bb55f71426642a2deaa787760
